toc.js

```diff
@@ -28,7 +29,7 @@ class Item {
     constructor($el) {
         if ($el) {
             this.slug = $el.attr('id');
-            this.text = $el.text();
+            this.text = $el.text().trim();
```

Comment:
This was necessary to avoid trailling spaces in many cases (notably, markdown-it-anchor's permalinks add a space automatically between the permalink and the content)